lib/utils.js

```diff
@@ -258,6 +258,19 @@ exports.autoStoreData = function (req, res, next) {
     req.session.data = {}
   }
 
+  try {
+    let defaultsFilePath = path.join(__dirname, '/../app/defaults.js')
```

Comment:
Is this expected to set other defaults or just defaults for the autoDataStore? If the latter, I wonder if the name could be more specific.

Reply:
It's just for the session data defaults, I've renamed the file to be more explicit